wdio.conf.js

```diff
@@ -23,7 +23,10 @@ const config = {
   ],
 
   // Terra-toolkit services are 'visual-regression', AxeService, TerraService, SeleniumDockerService
-  services: wdioConf.config.services.concat([WebpackDevService]),
+  // services: wdioConf.config.services.concat([WebpackDevService]),
```

Comment:
Oops, when you move this to the bottom, you didn't remove this comment.

Reply:
Removed: https://github.com/cerner/terra-clinical/pull/272/commits/3e968067da3a39875c1b528dd8615f112b61fd72